src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Scroll from 'react-scroll';
+import { focusElement } from 'platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
```

Comment:
We should have a utility for this but can't remember where... @ncksllvn do you remember?

Reply:
Its in util/ui - Seems like there is a mixed-bag as far as who uses it.  Lots of people use the implementation they have here.

Reply:
Yeah this is default when we generate an advance form.